azure-ts-dynamicresource/index.ts

```diff
@@ -0,0 +1,86 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as azure from "@pulumi/azure";
+
+import { CDNCustomDomainResource } from "./cdnCustomDomain";
+
+const location = "West US";
+
+/** The custom domain host of the CDN endpoint. */
+export let cdnCustomDomainResource: CDNCustomDomainResource | undefined;
+
+// Create an Azure Resource Group
+const resourceGroup = new azure.core.ResourceGroup("resourceGroup", {
+    location: location,
+});
+
+// Create an Azure resource (Storage Account) so we can point the CDN endpoint to it.
+const storageAccount = new azure.storage.Account("storage-account", {
+    resourceGroupName: resourceGroup.name,
+    location: resourceGroup.location,
+    accountTier: "Standard",
+    accountReplicationType: "LRS",
+});
+
+// Create a Blob container in the storage account.
+const blobContainer = new azure.storage.Container("blob-container", {
+    resourceGroupName: resourceGroup.name,
+    storageAccountName: storageAccount.name,
+    containerAccessType: "blob"
+});
+
+const cdnProfile = new azure.cdn.Profile("cdn-profile", {
+    location: location,
+    resourceGroupName: resourceGroup.name,
+    sku: "Standard_Akamai",
```

Comment:
What are the implications of choosing Akamai instead of Microsoft?

Reply:
I'll add a link to https://docs.microsoft.com/en-us/azure/cdn/cdn-features.